init.sh

```diff
@@ -74,7 +74,7 @@ echo "Local & bblayers conf set for $machine"
 # bsp submodule layer
 declare -A bsparr
 
-bsparr+=( ["minnowboard"]="meta-intel" ["raspberrypi2"]="meta-raspberrypi" ["raspberrypi3"]="meta-raspberrypi" ["porter"]="meta-renesas" ["silk"]="meta-renesas" ["dragonboard-410c"]="meta-qcom" )
+bsparr+=( ["minnowboard"]="meta-intel" ["raspberrypi2"]="meta-raspberrypi" ["raspberrypi3"]="meta-raspberrypi" ["porter"]="meta-renesas" ["silk"]="meta-renesas" ["dragonboard-410c"]="meta-qcom" ["r-car-m3-starter-kit"]="renesas-rcar-gen3")
```

Comment:
@gunnarx good catch Gunnar. I didn't spot that when adding it to the script. 

Just as input to any script redesign meta-linaro provides various linaro developed technologies including the Linaro toolchain. As a result although I've added it is entirely conceivable that other ARM YBSPs that come along later will also make use of it and so could be considered as shared amongst various YBSPs. That said, I suppose it is possible that for some reason they may use different SHAs.